tests/plugins/triage/index.js

```diff
@@ -200,4 +200,44 @@ describe("triage", () => {
             expect(issueLabelReq.isDone()).not.toBeTruthy();
         });
     });
+
+    describe("When a pull request is opened", () => {
+        test("Adds the label if there are no labels present", async() => {
+            const issueLabelGetReq = nock("https://api.github.com")
+                .get("/repos/test/repo-test/issues/1")
```

Comment:
Should this say "pulls" instead of "issues"? (I thought I remembered some weirdness around issues and pulls both working in GitHub's APIs at one point, but...)

Reply:
I think pulls are considered to be a subset of issues in the API, so all the [label endpoints](https://developer.github.com/v3/issues/labels/) have "issues" URLs, and requests that work with issues will generally also work with pull requests.